app/scripts/modules/core/instance/instances.component.ts

```diff
@@ -121,6 +136,16 @@ export class InstancesComponent implements IComponentOptions {
   public controller: any = InstancesController;
 }
 
+export class InstancesWrapperComponent implements IComponentOptions {
+  public bindings: any = {
+    instances: '<',
+    highlight: '<'
+  };
+  public controller: any = InstancesController;
+  public template = `<instances instances="$ctrl.instances" highlight="$ctrl.highlight"/>`;
```

Comment:
Spoke too soon, it is needed so the angular2react has something to wrap. Since the instances component builds itself on the fly, there's nothing for React to wrap.